compiler/rustc_driver_impl/messages.ftl

```diff
@@ -1,5 +1,6 @@
 driver_impl_ice = the compiler unexpectedly panicked. this is a bug.
 driver_impl_ice_bug_report = we would appreciate a bug report: {$bug_report_url}
+driver_impl_ice_bug_report_internal_feature = using internal features is not supported and is expected to lead to internal compiler errors
```

Comment:
micro nit: maybe something like "usage of internal features is unsupported and know to cause internal compiler errors" ?

Reply:
I like that wording better, or we could also soften it a bit more "and known to cause internal compiler errors when used incorrectly". I'm happy if people use internal features as long as they know they're mostly on their own if they opt to do so.

Reply:
I have a slight preference towards "expected to" instead of "known to", because it's not just that we know that it's broken and don't care, we _want_ it to be broken and deliberately add `bug!()`s.